ops/lib.rs

```diff
@@ -452,7 +452,9 @@ fn codegen_u8_slice(core: &TokenStream2, idx: usize) -> TokenStream2 {
       Ok(b) => {
         // Handles detached buffers.
         let byte_length = b.byte_length();
-        let store = b.data() as *mut u8;
+        let data = b.data();
+        assert!(data.is_some());
```

Comment:
Why the manual assertion? `data.unwrap()` will panic too

Reply:
Good point